charts/secrets-store-csi-driver/README.md

```diff
@@ -34,6 +34,7 @@ The following table lists the configurable parameters of the csi-secrets-store-p
 | `linux.image.repository`                | Linux image repository                                                                                                                                                         | `registry.k8s.io/csi-secrets-store/driver`              |
 | `linux.image.pullPolicy`                | Linux image pull policy                                                                                                                                                        | `IfNotPresent`                                          |
 | `linux.image.tag`                       | Linux image tag                                                                                                                                                                | `v1.3.4`                                                |
+| `linux.image.digest`                       | Linux image digest, image pull from digest instead of tag if specified                                                                                                                                                                 | `""`                                                |
```

Comment:
Please revert changes to `charts` folder. All the yaml changes in `manifest_staging` will be promoted into the released charts folder during the release.